### ISSUE TITLE
List datasets for Data registry launch

### ISSUE THREAD

**Issue by hrubyjan** [open-contracting/data-registry#56]
There is a long list of spiders but my feeling is that we don't want to process all as spiders like `chile_compra_record` and `chile_compra_releases` maybe eventually produces the same set of compiled releases.

And there is plenty of such spiders like  `colombia` + `colombia_bulk` or `afghanistan_record` + `afghanistan_record_packages` + `afghanistan_releases` + `afghanistan_release_packages`

Or Mexico has 8 spiders and I'm not sure whether we want to process all, or some data are subset of another dataset etc.

Can you please give me or delegate this task to someone who can give me a list of spiders to be processed.

**Comment by yolile** [comment]
@hrubyjan yes, we have a filtered list of spiders, you can find it [here](https://docs.google.com/spreadsheets/d/1OmL53-_PLPgJZFqxx0iwHRbRvB597bKPo5Kl6vuA_7U/edit?ts=605b609c#gid=0) (all the ones with "yes" in the "In" column)

**Comment by jpmckinney** [comment]
There are also those with "low" but they are lower priority and can be postponed, since we might not have their descriptions ready anyway.
